aten/src/ATen/native/SortingUtils.h

```diff
@@ -40,7 +40,7 @@ void dim_apply(TensorList tensors, int64_t dim, Fn f) {
   });
 }
 
-// ensure we get good values and indices for kthvalue, mode, median
+// ensure we get good values and indices for kthvalue, mode
```

Comment:
yikes, this historic comment could use some work. Oh well.

Reply:
This will eventually be replaced with TensorIterator once it supports dim_apply.